src/ui/index.ts

```diff
@@ -1,37 +1,58 @@
 export { Text } from './text/text.component';
+export { TextProps } from './text/text.type';
 export { Animated } from './animated/animated.component';
+export { AnimatedProps } from './animated/animated.type';
 export { Tooltip } from './tooltip/tooltip.component';
-export { TooltipRef } from './tooltip/tooltip.type';
+export { TooltipRef, TooltipProps } from './tooltip/tooltip.type';
 export { Badge } from './badge/badge.component';
+export { BadgeProps } from './badge/badge.type';
 export { Overlay } from './overlay/overlay.component';
-export { OverlayRef } from './overlay/overlay.type';
+export { OverlayRef, OverlayProps } from './overlay/overlay.type';
 export { Fab } from './fab/fab.component';
+export { FabProps } from './fab/fab.type';
 export { Input } from './input/input.component';
+export { InputProps } from './input/input.type';
 export { Mention } from './mention/mention.component';
+export { MentionProps } from './mention/mention.type';
 export { Textarea } from './input/textarea.component';
+export { TextareaProps } from './input/textarea.type';
 export { Div } from './div/div.component';
+export { DivProps } from './div/div.type';
 export { Div as Box } from './div/div.component';
```

Comment:
Why change the name to Box?

Reply:
It exports both. `Box` is an alias of `Div`

![image](https://user-images.githubusercontent.com/5173987/102300797-31606580-3f1b-11eb-9c87-9a8f0cf6542b.png)
https://magnus-ui.com/docs/div/

Reply:
Oh sorry, my mistake! Believe I didn't even knew that this alias exists?